common/beerocks/btl/btl_broker.cpp

```diff
@@ -0,0 +1,302 @@
+    using Flags = InterfaceConfigurationRequestMessage::Flags;
+
+    uint32_t n = 0;
+    string_utils::copy_string(interface_configuration_request_msg.metadata()->interfaces[n].ifname,
+                              bridge_iface.c_str(), IF_NAMESIZE);
+
+    interface_configuration_request_msg.metadata()->interfaces[n].flags |= Flags::IS_BRIDGE;
+    n++;
+    THREAD_LOG(DEBUG) << "adding bridge " << bridge_iface
+                      << " to ieee1905 transport, bridge iface=" << bridge_iface;
+    for (const auto &iface : ifaces) {
+        string_utils::copy_string(
+            interface_configuration_request_msg.metadata()->interfaces[n].ifname, iface.c_str(),
+            IF_NAMESIZE);
+        string_utils::copy_string(
+            interface_configuration_request_msg.metadata()->interfaces[n].bridge_ifname,
+            bridge_iface.c_str(), IF_NAMESIZE);
+        interface_configuration_request_msg.metadata()->interfaces[n].flags |=
+            Flags::ENABLE_IEEE1905_TRANSPORT;
+        n++;
+        THREAD_LOG(DEBUG) << "adding interface " << iface << " to ieee1905 transport"
+                          << " bridge=" << bridge_iface;
+    }
+    interface_configuration_request_msg.metadata()->numInterfaces = n;
+    THREAD_LOG(DEBUG) << "numInterfaces=" << n;
+
+    return transport::messages::send_transport_message(*bus, interface_configuration_request_msg);
+}
+
+void transport_socket_thread::add_socket(Socket *s, bool add_to_vector)
+{
+    socket_thread::add_socket(s, add_to_vector);
+}
+
+void transport_socket_thread::remove_socket(Socket *s) { socket_thread::remove_socket(s); }
+
+bool transport_socket_thread::read_ready(Socket *s) { return socket_thread::read_ready(s); }
+
+bool transport_socket_thread::bus_subscribe(const std::vector<ieee1905_1::eMessageType> &msg_types)
+{
+    LOG_IF(!bus, FATAL) << "Broker socket not initialized!";
+
+    if (msg_types.size() > SubscribeMessage::MAX_SUBSCRIBE_TYPES) {
+        LOG(ERROR) << "Subscribing to " << msg_types.size()
+                   << " is not supported. Maximal allowed types: "
+                   << SubscribeMessage::MAX_SUBSCRIBE_TYPES;
+
+        return false;
+    }
+
+    // Build a subscription message
+    SubscribeMessage subscribe;
+    subscribe.metadata()->type = SubscribeMessage::ReqType::SUBSCRIBE;
+
+    subscribe.metadata()->msg_types_num = 0;
+    for (const auto &msg_type : msg_types) {
+        subscribe.metadata()->msg_types[subscribe.metadata()->msg_types_num].bits = {
+            .internal = 0, .vendor_specific = 0, .reserved = 0, .type = uint32_t(msg_type)};
+
+        ++subscribe.metadata()->msg_types_num;
+    }
+
+    return transport::messages::send_transport_message(*bus, subscribe);
+}
+
+bool transport_socket_thread::bus_connect(const std::string &beerocks_temp_path,
+                                          const bool local_master)
+{
+    return true;
+}
+
+void transport_socket_thread::bus_connected(Socket *sd) {}
```

Comment:
This can be deleted now - btl_uds shouldn't be needed anymore now that we have a proper zmq replacement.

Reply:
True, removed.